conf/capsule.yaml.template

```diff
@@ -1,4 +1,6 @@
 CAPSULE:
+  # Capsule hostname for N-minus testing
```

Comment:
hostname can be generally used, so comment can be removed?, and I don't see this setting added on CI conf, instead it is directly used in pipeline, so do we need it here?
```suggestion

```

Reply:
Since all conf templates have some information about setting, I added this. Keeping it is no harm.

Its good to keep setting in sync in CI and robottelo especially in confs, so that local execution of tests wont fail.

I will open an MR in jenkins to create the conf.